spec/unit/rest/Twilio.spec.js

```diff
@@ -0,0 +1,52 @@
+var proxyquire = require('proxyquire');
+var url = require('url');  /* jshint ignore:line */
+var Q = require('q');
+
+describe('hostname', function() {
+  var client;
+  beforeEach(function() {
```

Comment:
Can we use `nock` instead of a mock here? Recent example: https://github.com/sendgrid/sendgrid-nodejs/pull/1093/files#diff-0a22ccfa1c28c602273fbc87ebe73ac2R42